src/Exception/DeveloperDoesNotExistException.php

```diff
@@ -45,7 +45,7 @@ class DeveloperDoesNotExistException extends ApiException implements ApigeeEdgeE
    * @param \Throwable|null $previous
    *   Previous exception.
    */
-  public function __construct(string $email, string $message = 'Developer with @email email address not found.', int $code = 0, \Throwable $previous = NULL) {
+  public function __construct(string $email, string $message = 'Developer with @email email address not found. Running the developer sync could fix this.', int $code = 0, \Throwable $previous = NULL) {
```

Comment:
Okay, but only admins can run the developer sync... why we would suggest this to a simple authenticated user?